RickBacci/ideabox3
# remove comments and console.logs

#### What's this PR do?

refactor code to remove comments and console.logs
#### Where should the reviewer start?

Changes made to one file only.
#### How should this be manually tested?

You can update the idea body in one of the ideas to verify that the code still works.

What gif best describes this PR or how it makes you feel?

Feels good to refactor ;)

@egogre @rrgayhart
